editor/inserter/menu.js

```diff
@@ -330,7 +341,7 @@ export class InserterMenu extends Component {
 					ref={ this.bindReferenceNode( 'search' ) }
 					tabIndex="-1"
 				/>
-				<div role="menu" className="editor-inserter__content">
+				<div ref={ ( r ) => this._tabContainer = r } role="menu" className="editor-inserter__content">
```

Comment:
Do you think we can remove the `_`, we're not using this convention for the private properties?
Can we avoid small variables like `r` maybe `ref` ?